recipes/zpp/all/conanfile.py

```diff
@@ -0,0 +1,52 @@
+from conans import ConanFile, CMake, tools
+
+
+class HelloConan(ConanFile):
+    name = "zpp"
```

Comment:
```suggestion
    name = "zapproject-zpp"
```

Because http://zpp-library.sourceforge.net/ is older and more popular